rust-runtime/smithy-types/src/retry.rs

```diff
@@ -5,6 +5,8 @@
 
 //! This module defines types that describe when to retry given a response.
 
+use std::fmt::{Display, Formatter};
```

Comment:
can you add `#[warn(missing_docs)]` and add all the docs it asks for? we should get in the habit of having this at the crate level, but until then, we should do it at the module level especially for code that customers will end up touching